IPython/utils/tempdir.py

```diff
@@ -1,98 +1,8 @@
-"""TemporaryDirectory class, copied from Python 3.2.
-
-This is copied from the stdlib and will be standard in Python 3.2 and onwards.
```

Comment:
The module should still have a docstring. It actually grew a couple of other classes which are not in the stdlib, and those should be mentioned in the docstring too.